src/components/InstrumentControls/instrumentFields/CameraBinSelectField.vue

```diff
@@ -1,20 +1,20 @@
 <template>
   <b-field
     :horizontal="horizontal"
-    label="Bin"
+    label="Resolution"
   >
     <b-select
       v-model="localSelection"
-      placeholder="Select bin"
       size="is-small"
     >
       <option
-        v-for="(bin_option, index) in binModes"
-        :key="index"
-        :value="bin_option"
-        :selected="index === 0"
+        value="optimal"
```

Comment:
Just to be clear, the `"optimal"` and `"maximum"` strings get "translated" into a binning option that cameras understand in the observatory code, right?

Reply:
Yes, exactly that!